src/input-group.scss

```diff
@@ -49,8 +49,6 @@ $block: #{$fd-namespace}-input-group;
   .#{$block}__input {
     @include fd-reset-child-spacing();
 
-    flex: 1 0 auto;
```

Comment:
This was necessary for the implementation libraries (definitely fundamental-react), can you link this change and show it doesn't affect anything?

Reply:
If you remove this declaration, what are the `flex-grow`, `flex-shrink` and `flex-basis` values?

Reply:
As I could notice setting these properties makes the `__input` to take whole width of his parent, not matter if there is any sibling. (it's changed there by `width:auto`, but it's still not correct)
![image](https://user-images.githubusercontent.com/26483208/66136616-0e8aaf80-e5fc-11e9-87b4-bcf6ee658a78.png)
I dont know why you need it there, because by default, without `flex:...` it's gonna take whole width of parent minus the size of addon.